bitcoin/pullpush.c

```diff
@@ -1,3 +1,5 @@
+#include <assert.h>
```

Comment:
CI complains about this, it should be before the `#include <ccan/endian/endian.h>`